ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/TomlStubEmptyNode.kt

```diff
@@ -14,4 +15,10 @@ public class TomlStubEmptyNode(lineNo: Int, config: TomlConfig = TomlConfig()) :
     config
 ) {
     override val name: String = EMPTY_TECHNICAL_NODE
+
+    override fun write(
+        emitter: TomlEmitter,
+        config: TomlConfig,
+        multiline: Boolean
+    ) { }
```

Comment:
strange that diktat is not raising a warning here, could you please add a comment inside this block, something like: `no need to write anything for a stub node`?